datahub-web-react/src/graphql/dataset.graphql

```diff
@@ -331,7 +331,7 @@ fragment getRoles on Access {
             }
             urn
             actors {
-                users {
+                users(input: $actorUserUrnFilter) {
```

Comment:
then if we add the new `isAssignedToMe` field, we can update this query to not even fetch the `users` here since it looks like right now the UI is only getting `users` to check if the logged in user is in the list. So that would be a big performance boost